### ISSUE TITLE
Launch plan for v2

### ISSUE THREAD

**Issue by asood123** [opencollective/opencollective#431]
Eng tasks (Xavier)
- [X] Make sure we check for customerId in its own column as well as `data`
- [ ] Add an api test for existing stripe flow (meaning, customerId is still on the Host only and the webhook should still work)
- [x] Add a frontend test for donation flow (mocking calls to stripe and api)
- [x] Frontend changes to match existing ones
   - [x] nicer looking counter for annual budget
   - [x] highlight contribute buttons
   - [x] Need Submit Expense button
   - [x] Clicking on `paid` under Expense shrinks to nothing. Test on Webpack
- [x] Bug: Credit card field shows "other" when nothing there
- [x] Bug: says "latest transaction" on both columns. First one should say 'latest expenses'
- [x] Show Events on the collective page
- [x] available funds are now missing. 
- [x] why is trivago not a gold sponsor? (it already gave +10k)
- [x] events:
   - [x] if try to create an event not logged in I get an internal server error. Expected behaviour: message with a log in requirement, or feature available to members only
   - [x] events/new is [not loading](https://staging.opencollective.com/betweenthewires/events/new) 
   - [x] existing event [not loading](https://staging.opencollective.com/opensource/events/webpack-webinar-83ev)
  - [x] when click in create event (logged in) there's a 404 and then loads the event
  - [x] event background image [not loading](https://staging.opencollective.com/opensource/events/webpack-webinar-83ev)
  - [x] getting an error when saving a new event 'you need to define a URL for your event' but no where to create one from? [see](https://staging.opencollective.com/railsgirlsatl/events/new)
   - [x] getting an error when saving a new event. Nothing happens after I hit 'create'. error in console reads: "TypeError: e.tiers.map is not a function"
   - [x] create new event, can't save (because of two errors, see screenshots in slack). Events are created with an old date. [see](https://staging.opencollective.com/opensource) 
   - [x] problem with saving new event with edited tiers. (see slack)
- [x] expenses
     - [x] add funds is missing from [/expenses](https://staging.opencollective.com/betweenthewires/expenses) 
     - [x] collective's balance [shows negative](https://staging.opencollective.com/opensource/expenses)
     - [x] [can't see expenses](https://staging.opencollective.com/webpack/expenses) when not 
        logged in. see [this](https://staging.opencollective.com/betweenthewires#budget) vs [this](https://staging.opencollective.com/betweenthewires/expenses)
- [x]  ‘unexpected error’ when logged in and clicking in ‘all transactions’
- [x] description format breaks see [here](https://staging.opencollective.com/react-boilerplate) vs [here](https://opencollective.com/react-boilerplate) 
- [x] organization is showing with title 'backer' in Sponsors list [see](https://staging.opencollective.com/wordpress-sf) and [here](https://staging.opencollective.com/railsgirlsatl)
- [x] organization still showing as a backer in [this collective](https://staging.opencollective.com/wordpress-sf) and also [here](https://staging.opencollective.com/vuejs)
- [x] subscriptions is not loading [see](https://staging.opencollective.com/subscriptions)
- [x] subscriptions fail to cancel, need to update controller to account for new customerId location. 
- [x] profiles:
   - [x] new profile created when making a donation is serving a 500. See [here](https://staging.opencollective.com/piamancini2)
   -  [x] new profile created when making a donation keeps timeouts at loading. see image in #engineer 
- [x] organizations:
   - [x] when logged in with a new user, I try to pay as an organization and system is not showing a way to add a new credit card to the profile. I can make a donation as an organization with the credit card saved in my suer profile but I can't add a new one. 
   - [x]  created organization from users profile, added new users, users don't show up in the organization's profile. See [here](https://staging.opencollective.com/organization1/)
   - [x] when adding members, redirect not working (maybe it's just in staging) and no easy way to toggle back to organization's profile to see changes. From [here](https://staging.opencollective.com/organization1/edit#info)
- [x] tried a credit card with a wrong expiry date. no error served. screenshot in slack. (this is hapenning still)
- [x] Collectives not loading. See [here](https://staging.opencollective.com/react-boilerplate), [here](https://staging.opencollective.com/wwcodeaustin).
- [x] available balance still not showing.
- [x] donation flow
  - [x] When I click on a sponsor tier, the donation interface is the default one, it should pre-fill the tier settings. [See for example](https://staging.opencollective.com/webpack/order/266) I see that when hitting silver sponsor tier.  

Database-tasks
- [x] Remove any unnecessary custom tiers in database 

Test & Verification
- [x] Test logged in one-time and recurring donations (Aseem)
- [x] Test logged out one-time and recurring donations (Aseem)
- [ ] Test to download the invoice
- [x] Test that expenses can still be approved and paid
- [x] Test /create
- [x] Test /opensource/apply
- [x] Review all collectives with custom tiers and make sure they look ok (Pia)

Communications
- [x] preannouncement email (Pia)
- [x] Give a special heads up to the collectives with custom templates (like grav) (Pia)
- [x] blogpost announcing some changes (Pia)

@piamancini @xdamman

**Comment by piamancini** [comment]
Post-migration
- [ ] check and update slugs for Webpack tiers (Xavier)
- [ ] check three collective -> collective txns and delete rows manually. 35927 ($210), 35935 ($70), and 36534 ($8.64)

Post-launch (not blockers but must be done before end of month)
- [ ] update monthly mailing list scripts (Xavier)
- [ ] update check_stripe_subscriptions script (Aseem)
- [ ] Add a link to export to CSV
- [ ] fix spanish version
- [ ] annual budget on opencollective.com jumped. check why. 
- [ ] check # of collectives jumped on front page
- [ ] opencollective.com/event/new not loading 

Post-launch (nice to haves)
 - [ ] after donation, give users the next prompt (like check your email). Currently, we take them to profile page with nothing to do next and they aren't logged in
- [ ] how we are managing sponsors after one year
- [ ] how sponsors jump category because of accumulated amount
- [ ] flip tiers order

**Comment by nicobarretoOC** [comment]
- [ ] When editing profile and adding website address, clicking save does not store it.

- [ ] Donation amount changes when checking out (email from Harri).

**Comment by xdamman** [comment]
The idea of a simple contributor is to add someone to the organization without giving them the permission to approve an expense. Especially important for non Github based collectives.

**Comment by piamancini** [comment]
Great work everyone!!!!! 
Closing this issue. We've opened separate issue for remaining things and tagged them V2. 
Also created #478 and #502 to follow up on v2 expenses and design.
